controllers/numaresourcesscheduler_controller.go

```diff
@@ -40,6 +41,11 @@ import (
 	"github.com/openshift-kni/numaresources-operator/pkg/status"
 )
 
+const (
+	defaultNUMAResourcesSchedulerCrName                      = "numaresourcesscheduler"
```

Comment:
this doesn't seem needed but it also doesn't seem wrong, and allow us to test a bit better, so let's keep it